src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import 'header';
+@import 'variables';
```

Comment:
According to the checklist, the variables should be declared before any other styles. So, you should import your variables first before importing other scss files.